casper/src/test/scala/coop/rchain/casper/api/BlockQueryResponseAPITest.scala

```diff
@@ -59,10 +59,11 @@ class BlockQueryResponseAPITest
       setValidator = sender.some,
       setDeploys = randomDeploys.some,
       setJustifications = List(Justification(bondsValidator.validator, genesisBlock.blockHash)).some,
+      setParentsHashList = List(genesisBlock.blockHash).some,
       setBonds = List(bondsValidator).some
     )
 
-  val faultTolerance = 1f
+  val faultTolerance = -1f
```

Comment:
Should this be named `faultToleranceMin`? Or definition from `MIN_FAULT_TOLERANCE` from SafetyOracle can be used.

Reply:
I don't think so, it a fault tolerance that is expected in tests. It just should be minimal

Reply:
I think here it is just testing that block response for block query API is valid. Whether returned fault tolerance is minimal or some else - out of scope. It just happen that given how test is defined, fault tolerance return is -1.